packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -356,13 +411,10 @@ export class CfnInclude extends core.CfnElement {
     } else {
       const l1ClassFqn = cfn_type_to_l1_mapping.lookup(resourceAttributes.Type);
       if (l1ClassFqn) {
-        const options: core.FromCloudFormationOptions = {
-          finder,
-        };
```

Comment:
Can you leave this? I like this small piece of type safety.

Of course, replace `finder` with `parser: cfnParser`.